Style.md

```diff
@@ -67,11 +55,15 @@ import (
 
 ### Exceptions
 
-As recommended by go modules version design,
-in the example of `"github.com/go-redis/redis/v7"`,
-`v7` is the major version number,
-so we consider `redis` as the actual last element of import path.
-As a result there's no need to rename import it.
+The implied package name of all of the following is still "foo", and thus
+they do not need to be renamed:
+
+* `.../foo.v1`
+* `.../foo/v1`
+" `.../foo-go`
+" `.../foo.go`
```

Comment:
`"` should be `*` instead (the current markdown rendering is wrong: https://github.com/reddit/baseplate.go/blob/8817556713b648bed2c4bf1fd844265b01546c50/Style.md#exceptions-1)